### PR TITLE
fix: Encode path parameters

### LINKED ISSUE #102: Path parameters are not being url encoded
Hi,

I just noticed that both path parameters are not being url encoded.

My suggestion would be to encode each path parameter with `encodeURIComponent`.

What do you think?

EDIT: Discovered that query were already encoded.

### PR DESCRIPTION
**Pull request by rtfpessoa**
Fixes #102

### CODE REVIEW

File: src/runtime/query.ts [hunk 1]
```diff
@@ -29,7 +29,9 @@ export function deep(
         if (typeof v === "object") {
           return visit(v, key);
         }
-        return qv`${key}=${v}`;
+        // TODO: Check if va can be anything else than string
+        // @ts-expect-error How can we know that v needs to be a string?
```
Comment (rtfpessoa) [inline src/runtime/query.ts:33]: Not sure how to deal with this? Any suggestions?

**Comment by rtfpessoa** [comment]
@jonkoops @chrisvanmook @fgnass can you please take a look in here?

**Comment by jonkoops** [comment]
Sorry, I am not a regular maintainer of this project (I just do chores like keeping the dependencies up-to-date).

**Review by shocklateboy92 — APPROVED** [review]
This change seems sensible.
@fgnass let me know if you have any objections. Otherwise, I'll merge it in a few days.

**Pull request merged**